index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Posh UI</title>
+
+    <!-- favicon -->
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="assets/brand-logos/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="assets/brand-logos/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="assets/brand-logos/favicon-16x16.png"
+    />
+
+    <!-- poshui-components cdn -->
+    <link
+      rel="stylesheet"
+      href="https://poshui-components.netlify.app/css/main.css"
+    />
+
+    <!-- poshui-documentation css -->
+    <link rel="stylesheet" href="app.css" />
+  </head>
+  <body>
+    <div class="container margin-auto">
+      <header class="header">
+        <img
+          class="img-brand-logo"
+          src="assets/brand-logos/brand-logo.png"
+          alt="brand logo png"
+        />
+        <a class="link-brand margin-lr-1" href="./index.html">Posh UI</a>
+
+        <nav class="margin-l-auto navbar">
+          <ul class="list-no-bullets">
+            <li class="nav-link margin-lr-1">
+              <a href="./index.html">Home</a>
+            </li>
+            <li class="nav-link margin-lr-1">
+              <a href="#">Documentation</a>
+            </li>
+            <li class="nav-link margin-lr-1">
+              <a
+                href="https://github.com/hsnice16/PoshUI-Documentation"
+                target="_blank"
+                >GitHub</a
+              >
```

Comment:
Closing tag '>' indent has been separated to a new line.

Reply:
May be, It is because of prettier. But thanks for pointing it, Will look into it.

Reply:
I checked this, prettier is doing. So, I think, I should let it be like this.